src/ngx_http_upstream_jdomain_module.c

```diff
@@ -117,7 +117,7 @@ ngx_module_t ngx_http_upstream_jdomain_module = { NGX_MODULE_V1,
 	                                                NULL,                                  /* exit master */
 	                                                NGX_MODULE_V1_PADDING };
 
-static struct sockaddr_in NGX_JDOMAIN_INVALID_ADDR_SOCKADDR_IN = {};
+static struct sockaddr_in NGX_JDOMAIN_INVALID_ADDR_SOCKADDR_IN = {0};
```

Comment:
I know it is petty, but linter is complaining about style. Please add spaces.

```suggestion
static struct sockaddr_in NGX_JDOMAIN_INVALID_ADDR_SOCKADDR_IN = { 0 };
```